<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reportPage</class>
 <widget class="QWidget" name="reportPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчёты</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblExaroPath">
     <property name="text">
      <string>Внешний редактор отчетов Exaro</string>
     </property>
     <property name="buddy">
      <cstring>edtExaroEditor</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="edtExaroEditor"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnSelectDocumentEditor">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtExtGenRep"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="btnSelectExaroEditor">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDocumentEditor">
     <property name="text">
      <string>Внешний редактор &amp;документов</string>
     </property>
     <property name="buddy">
      <cstring>edtDocumentEditor</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="edtDocumentEditor"/>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="btnSelectExtGenRep">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblExtGenRep">
     <property name="text">
      <string>Внешний генератор &amp;отчетов</string>
     </property>
     <property name="buddy">
      <cstring>edtExtGenRep</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtDocumentEditor</tabstop>
  <tabstop>btnSelectDocumentEditor</tabstop>
  <tabstop>edtExtGenRep</tabstop>
  <tabstop>btnSelectExtGenRep</tabstop>
  <tabstop>edtExaroEditor</tabstop>
  <tabstop>btnSelectExaroEditor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
